package.json

```diff
@@ -43,14 +43,12 @@
   },
   "dependencies": {
     "@stoplight/json": "^3.10.0",
-    "@stoplight/json-schema-merge-allof": "^0.7.2",
+    "@stoplight/json-schema-tree": "1.0.0-beta.5",
     "@stoplight/react-error-boundary": "^1.0.0",
     "@stoplight/tree-list": "^5.0.3",
-    "classnames": "^2.2.6",
-    "lodash": "^4.17.15"
```

Comment:
What, no lodash? I can't believe that, how do you even write code? 😇

Reply:
IIRC lodash was used in 2-3 spots in JSV. In fact, the only useful fn was `pick`, but I got rid of it in JST completely.

Reply:
I was just kidding 😄 It's nice that you got rid of it. It's still a transient dep - and likely will be for the foreseeable future - but less direct dependencies are always nice.

Reply:
Yeah, I'm not a fan of bringing the whole library if you only use it once or twice. :laughing: 
It's totally okay if you actually use it, but makes little to no sense if you only use `pick` in a single spot.

Reply:
True, especially because it doesn't tree shake as well & as easily as it should. At least it didn't about a year ago, as it's mainly CJS